opsdroid/parsers/crontab.py

```diff
@@ -14,7 +14,7 @@
     """Parse all crontab skills against the current time."""
     while opsdroid.eventloop.is_running():
         await asyncio.sleep(60 - arrow.now().time().second)
-        _LOGGER.debug(_("Running crontab skills"))
+        _LOGGER.debug(_(time.asctime(time.localtime(time.time()))))
```

Comment:
I don't think you need to call time as much as you are.

Could you explain why you haven't done `time.asctime()` or `time.asctime(time.localtime())`.

I am also keen not to lose the original message. 

```suggestion
        _LOGGER.debug(_("Running crontab skills at {}").format(time.asctime(time.localtime(time.time()))))
```

Reply:
Actually - I tried several  options and liked the formatting of this one - it is quite possible that your option is more efficient. If you look at my profile, this is the first time I am committing to an open source project.

As for the original message, as I understood the ticket that message was supposed to get lost?

Reply:
Welcome to open source, it's awesome to have you here 🎉! This kind of code review is pretty standard for open source contribution. Please don't be put off by the questioning, it's not a criticism, I'm just keen to understand why you have made these decisions.

Particularly I'm keen to understand the difference between :

- `time.asctime(time.localtime(time.time()))`
- `time.asctime(time.localtime())`
- `time.asctime()`

We should aim to add the least code possible to meet the requirement. Do the nested calls add value? I can imagine that localtime might help us with timezones. 

As for the original message, that's probably my fault for not being clear in the issue. I would like to keen the original message and also add the time to it.